data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	swizzle 6
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25100000
+		"shields" 37200
+		"hull" 11780
+		"required crew" 62
+		"bunks" 70
+		"mass" 725
+		"drag" 9.5
+		"heat dissipation" .4
+		"fuel capacity" 500
+		"outfit space" 872
+		"weapon capacity" 333
+		"engine capacity" 150
+		weapon
+			"blast radius" 500
+			"shield damage" 5000
+			"hull damage" 2500
+			"hit force" 3500
+	outfits
+		"Large Heat Shunt" 2
+		"Jump Drive"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Quantum Keystone"
+		"Geode Reactor"
+		"White Sun Reactor"
```

Comment:
Remember the Hai need to plunder to get these outfits. The Sunbeam Beetle had no outfits larger than 102 tons since that is all you can fit in a Shield Beetle with 2 outfits expansions.

Also, three boulder reactors took less space in total, once you consider cooling and power requirements, but that may not be true with your hull. The design must have enough power to fight non-stop, without relying on batteries.

Reply:
> Remember the Hai need to plunder to get these outfits. The Sunbeam Beetle had no outfits larger than 102 tons since that is all you can fit in a Shield Beetle with 2 outfits expansions.

You dont need two expansions to disable an enemy ship, numbers can do that too

Not sure about the boulders honestly, and do remember sunbeams consume a lot of power + I added a lot of batteries and they got buffed
I think we should both try that one mission using both designs

Reply:
In my tests, I'm removing the "102 ton pillaged outfit" cap because it reduces options by too much.

Reply:
There is also a better plunderer option planned for the Uhai in the Mantis, which won't be seen or available to player until invasion ramps up but would have still been around and able to be used.

Reply:
I don't think that's an option since they wouldnt risk the prototype, if those are actually in before getting wanderer shipyards and not just schematics.
However, if they got raided they can use water bugs to plunder the disabled raider, and even if it isnt the case they can modify shield beetles to hold more cargo easily.